packages/frontend/src/components/common/ResourceView/ResourceViewList/index.tsx

```diff
@@ -352,7 +329,10 @@ const ResourceViewList: FC<ResourceViewListProps> = ({
     }, [items, columnSorts, columns]);
 
     return (
-        <Table className={classes.root} highlightOnHover>
+        <Table
+            className={`${classes.root} ${classes.smallHeaderText}`}
```

Comment:
`useTableStyles` also exports `cx` utility that you can use

`className={cx(classes.root, classes.smallHeaderText)}`